openedx_webhooks/lib/github/test/models/test_github_webhook_event.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import arrow
+import pytest
+
+from openedx_webhooks.lib.github.models import GithubWebHookEvent
```

Comment:
This is how I think of it: Use relative imports when files are intra-package, absolute imports otherwise. Tests are not part of the package, so should use absolute imports.